src/components/Login/LoginHeader.js

```diff
@@ -2,23 +2,29 @@ import Header from "../Reusable/Header"
 import NavBar from "../Reusable/NavBar"
 import Icon from "../Reusable/Icon"
 import Text from "../Reusable/Text"
+import Button from "../Reusable/Button"
+import { useTheme } from "../../context/theme-context"
+import { getTextColor } from "../../utils/theme-util"
 
 const LoginHeader = () => {
+    const { theme, toggleTheme } = useTheme()
 
     return (
         <Header>
 
-            <Text classes='txt-ucase txt-primary txt-lg' text='sneakerstore' />
+            <Text classes={`txt-ucase ${getTextColor(theme)} txt-primary txt-lg`} text='sneakerstore' />
```

Comment:
Nicely done. Everything is converted to a Component even "Text".  But you may get into issue in future. If you have to style text differently eg. including bold text somewhere, increasing fontsize of a particular character etc.

Reply:
thanks @VLeads. For the case mentioned, I have dedicated classes in my component library. So I can pass those classes to make the changes.